src/test/spec/oidc.rs

```diff
@@ -1286,3 +1285,25 @@ mod gcp {
         Ok(())
     }
 }
+
+mod k8s {
+    use crate::{
+        bson::{doc, Document},
+        Client,
+    };
+
+    // There's no spec test for K8s, so we run this simple sanity check.
+    #[tokio::test]
+    async fn successfully_authenticates() -> anyhow::Result<()> {
+        get_env_or_skip!("OIDC");
+
+        let client = Client::with_uri_str(mongodb_uri_single!()).await?;
```

Comment:
`mongodb_uri_single` looks for the "MONGODB_URI_SINGLE" environment var and skips if it's unset - where is this variable being defined? I can't seem to find it anywhere

Reply:
It's set in the drivers-evergreen-tools [k8s/setup.sh](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/f8420d2cf735a1d574e9be48428ac481c5572688/.evergreen/auth_oidc/k8s/setup.sh#L16) script.